src/sentry/lang/native/applecrashreport.py

```diff
@@ -221,19 +271,17 @@ def get_thread_apple_string(self, thread_info):
         return thread_string + "\n".join(rv)
 
     def _convert_frame_to_apple_string(self, frame, next=None, number=0):
-        if frame.get("instruction_addr") is None:
-            return None
-        slide_value = self._get_slide_value(frame.get("image_addr"))
-        instruction_addr = slide_value + self._parse_addr(frame.get("instruction_addr"))
-        image_addr = slide_value + self._parse_addr(frame.get("image_addr"))
+        frame_instruction_addr = frame["instruction_addr"]
+        frame_image_addr = frame.get("image_addr", 0)
+        slide_value = self._get_slide_value(frame_image_addr)
+        instruction_addr = slide_value + frame_instruction_addr
+        image_addr = slide_value + frame_image_addr
         offset = ""
         if frame.get("image_addr") is not None and (
             not self.symbolicated
             or (frame.get("function") or NATIVE_UNKNOWN_STRING) == NATIVE_UNKNOWN_STRING
         ):
-            offset = " + %s" % (
-                instruction_addr - slide_value - self._parse_addr(frame.get("symbol_addr"))
-            )
+            offset = " + %s" % (instruction_addr - slide_value - frame.get("symbol_addr", 0))
```

Comment:
Nit: F-strings are more understandable in this case:
```suggestion
            offset_value = instruction_addr - slide_value - frame.get("symbol_addr", 0)
            offset = f" + {offset_value}"
```